lib/remote/apilistener.cpp

```diff
@@ -329,6 +334,31 @@ void ApiListener::RenewOwnCert()
 	UpdateSSLContext();
 }
 
+void ApiListener::RenewCA()
+{
+	auto certPath (GetCaDir() + "/ca.crt");
+	auto cert (GetX509Certificate(certPath));
+
+	if (IsCertUptodate(cert)) {
+		return;
+	}
+
+	Log(LogInformation, "ApiListener")
+		<< "Our CA will expire soon, but we own it. Renewing.";
+
+	cert = RenewCert(cert);
```

Comment:
That function doesn't look like it would create a valid CA certificate as it calls `CreateCertIcingaCA()` which in turn calls `CreateCert()` with `ca=false`.

Reply:
I didn't conceal this hasn't been tested, yet. But thanks for pointing out, you likely saved me some headache! 👍